### ISSUE TITLE
`xreader.desktop` MIME type syntax error

### ISSUE THREAD

**Issue by spamegg1** [linuxmint/xreader#632]
### Distribution

Mint 21.2

### Package version

3.8.5

### Frequency

Always

### Bug description

`/usr/share/applications/xreader.desktop` has a syntax error in its MIME type. 

Towards the end, there is 
```
application/x-ext-cbzapplication/oxps
```
but it's missing a semicolon, it should be:
```
application/x-ext-cbz;application/oxps
```

### Steps to reproduce

1. Open `/usr/share/applications/xreader.desktop` in a text editor.
2. See bug (at the very bottom)

### Expected behavior

File should have correct syntax.

### Additional information

Bug encountered by a new forum user here: https://forums.linuxmint.com/viewtopic.php?p=2413157#p2413157

**Comment by kardell1** [comment]
si , precisamente ese era el error y la solucion al problema

**Comment by kludikovsky** [comment]
Identical effect here.

Might come from another app (application/x-ext-cbz) by not properly adding the MIME-type (missing semicolon at the end). But this is just a speculation.

Mint 21.2

**Comment by mtwebster** [comment]
Hi,, this is fixed in 4.0.1:
https://github.com/linuxmint/xreader/commit/495835ec8403eddd85f05c836cb04b2b3de34a7f
